### PR TITLE
New userspace and keymap added

### PR DESCRIPTION
**Pull request by edndn**
Added my personal keymap for `keebio/foldkb`.
The keymap has..

* Key layout similar to hhkb for my own compatibility: e.g. `ctrl` key at the mid row
* Custom rotary encoder: toggling between two modes (volume up/dn OR page up/dn)
* Custom tapping behaviors (tapdance)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/foldkb/keymaps/edndn-ctrlmidrow/keymap.c [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    _______,          KC_CAPS, KC_BTN1, KC_MS_U, KC_BTN2, _______, _______,     _______, _______, KC_PSCR, KC_SLCK, KC_PAUS, KC_UP,   _______, KC_DEL,
+    _______,          _______, KC_MS_L, KC_MS_D, KC_MS_R, _______, _______,     _______, _______, _______, _______, KC_LEFT, KC_RGHT,          _______,
+    _______,          _______, _______, _______, _______, _______, _______,     _______, _______, _______, _______, KC_DOWN, KC_DOWN, _______,
+    _______,          _______, _______, _______, _______, _______, _______,     _______, _______, _______, _______, C_EMOJI, _______
+  )
+};
+   
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    if (index == 0) {
+        switch(biton32(layer_state)){
```
Comment (drashna) [inline keyboards/keebio/foldkb/keymaps/edndn-ctrlmidrow/keymap.c:62]:
```suggestion
        switch(get_highest_layer(layer_state)){
```

File: keyboards/keebio/foldkb/keymaps/edndn-ctrlmidrow/rules.mk [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+CONSOLE_ENABLE = yes
+LTO_ENABLE = yes
+MOUSEKEY_ENABLE = yes
+EXTRAFLAGS += -flto
```
Comment (drashna) [inline keyboards/keebio/foldkb/keymaps/edndn-ctrlmidrow/rules.mk:5]: `LTO_ENABLE` already does this, no reason to do it twice. 
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/keebio/foldkb/keymaps/edndn-ctrlmidrow/keymap.c (hunk 1 above)
Comment (edndn) [reply keyboards/keebio/foldkb/keymaps/edndn-ctrlmidrow/keymap.c:62]: Oh, thank you for pointing this out!

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,10 @@
+# Ignore every folders in keyboards exepct few that i use
+
+/keyboards/*
+!/keyboards/keebio/
+
+
+
```
Comment (fauxpark) [inline .gitignore]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (edndn) [reply .gitignore]: Sorry my mistake.. I editted wrong project folder. Thank you for fixin it

**Pull request closed without merging**